protocol-units/bridge/move-modules/sources/atomic_bridge_counterparty.move

```diff
@@ -72,6 +72,25 @@ module atomic_bridge::atomic_bridge_counterparty {
         move_to(resource, bridge_transfer_store);
         move_to(resource, bridge_config);
     }
+
+    #[view]
+    public fun bridge_transfers(bridge_transfer_id : vector<u8>) : BridgeTransfer acquires BridgeTransferStore, BridgeConfig {
+        let config_address = borrow_global<BridgeConfig>(@atomic_bridge).bridge_module_deployer;
+        let store = borrow_global<BridgeTransferStore>(config_address);
+        if (!aptos_std::smart_table::contains(&store.transfers, bridge_transfer_id)){
+            BridgeTransfer {
+                amount: 0,
+                originator: @atomic_bridge,
+                recipient: vector::empty<u8>(),
+                hash_lock: vector::empty<u8>(),
+                time_lock: 0,
+                state: 0,
```

Comment:
If there is no `BridgeTransfer` for its key `bridge_transfer_id`, we would want to return an error here, instead of creating a zeroed `BridgeTransfer`.

Reply:
if changing to error.
```
#[view]
    public fun bridge_transfers(bridge_transfer_id : vector<u8>) : BridgeTransfer acquires BridgeTransferStore, BridgeConfig {
        let config_address = borrow_global<BridgeConfig>(@atomic_bridge).bridge_module_deployer;
        let store = borrow_global<BridgeTransferStore>(config_address);
        if (!aptos_std::smart_table::contains(&store.transfers, bridge_transfer_id)){
            assert!(false, 1);
        } else {
            let bridge_transfer = aptos_std::smart_table::borrow(&store.transfers, bridge_transfer_id);
            *bridge_transfer
        }
    }
```
    This errors with an error that indicates that I should return a BridgeTransfer
```
    transfers: SmartTable<vector<u8>, BridgeTransfer>,
   │                                             -------------- Found: '(atomic_bridge=0xC3BB8488AB1A5815A9D543D7E41B0E0DF46A7396F89B22821F07A4362F75DDC5)::atomic_bridge_initiator::BridgeTransfer'. It is not compatible with the other type.
   ·  
87 │ ╭         if (!aptos_std::smart_table::contains(&store.transfers, bridge_transfer_id)){
88 │ │             assert!(false, 1);
   │ │                              - Found: '()'. It is not compatible with the other type.
89 │ │         } else {
90 │ │             let bridge_transfer = aptos_std::smart_table::borrow(&store.transfers, bridge_transfer_id);
91 │ │             *bridge_transfer
92 │ │         }
   │ ╰─────────^ Incompatible branches
   ```

Reply:
I've updated it to abort (see this commit: https://github.com/movementlabsxyz/movement/pull/434/commits/d90348c008adf232096186850f5d2f99aa6ff6b0)